pkg/jobrunaggregator/jobrunaggregatorlib/util.go

```diff
@@ -45,6 +45,11 @@ func WaitAndGetAllFinishedJobRuns(ctx context.Context,
 	variantInfo string) ([]jobrunaggregatorapi.JobRunInfo, []jobrunaggregatorapi.JobRunInfo, []string, []string, error) {
 	clock := clock.RealClock{}
 
+	// Polling involves some queries that can add up to substantial costs, there's little
```

Comment:
Initial wait should be changed here:

https://github.com/openshift/ci-tools/blob/8702a80dfffe2c8455ddd9787a74821d41cf1369/pkg/jobrunaggregator/jobruntestcaseanalyzer/analyzer.go#L485

And here:

https://github.com/openshift/ci-tools/blob/8702a80dfffe2c8455ddd9787a74821d41cf1369/pkg/jobrunaggregator/jobrunaggregatoranalyzer/analyzer.go#L73

Reply:
heh